src/neptune/new/internal/init/project.py

```diff
@@ -13,6 +13,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+__all__ = ["init_project", "get_project"]
```

Comment:
Do we want to add deprecated funcitons like `get_project`?

Reply:
Yes at this point